src/utils/Api.js

```diff
@@ -7,6 +7,14 @@ export default class Api {
       .catch(e => e);
   }
 
+  static fetchAlerts() {
+    return fetch(
+      'http://localhost:8000/alerts',
```

Comment:
Please update your code here and anywhere else where you have hardcoded the backend url to use an environment variable, such as API_URL , which can be set to http://localhost:8000 locally and to the appropriate url remotely